package.json

```diff
@@ -10,8 +10,11 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "hash-router": "^0.4.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router": "^5.2.1",
```

Comment:
Can you check if this `react-router` is required to be installed? I don't this this is required.

Please [refer this video](https://reactrouter.com/)